public/scripts/app.js

```diff
@@ -47,67 +39,37 @@ $('ul.nav li').click(function(){
          `
        );
      }
-   }
-   else if(($(this).context.textContent)==="My Work")
-   {
-     //projects section
-
+    } else if(($(this).context.textContent)==="My Work") {
      $("#addProject").fadeIn();
      $("#addVacation").fadeOut();
-
-     console.log("Projects section");
      $.ajax({
        method:"GET",
-       url:'/api/profile/projects',
+       url:'/api/profile',
        success:function(data){
-         console.log(data);
-         $p_id=data[0]._id;
-         console.log($p_id);
-         $("#results").empty();
-
-         console.log(data[0].projects.length);
-         renderAllProjects(data);
-
+         renderAllProjects(data[0]);
        },
        error: function(err){
-         console.log("ajax call failed for My Work",err);
-       }
-     })
-     var link=`/api/profile/${$p_id}/projects/:_id`
-     $.ajax({
-       method:'GET',
-       url:link,
-       success:function(data){
-         console.log(data);
-       },
-       error:function(err){
-         console.log(err);
+         console.log("ajax call failed for About Section",err);
        }
-     })
-   }
-   else if(($(this).context.textContent)==="Vacation")
-   {
+      });
+   } else if(($(this).context.textContent)==="Vacation") {
     $("#addProject").fadeOut();
     $("#addVacation").fadeIn();
 
-    console.log("Vacation section");
     $.ajax({
       method:"GET",
-      url:`/api/profile/${$p_id}/vacation`,
+      url:`/api/profile/${$profile_id}/vacation`,
       success:function(data){
-        console.log(data);
-        $("#results").empty();
```

Comment:
Moved `$("#results").empty();` to to `renderAllVacation`, for previously given reason.